### DISCUSSION TITLE
v5 release plan

### DISCUSSION THREAD

**Discussion by ascensus-mdb** [twbs/bootstrap#30610] [General]
Very good idea about GitHub discussions!

First questions - is it possible to plan and then to provide any, even very approximate dates for subsequent releases v.5? I think it would be extremely useful for the entire Bootstrap community.

And thanks for all hard, great work to the entire bootstrap team. You guys rock!

**Comment by dawidadach** [comment]
That's great idea - full support!

**Comment by mdo** [comment]
Yes, agreed! We're trying to line up v4.4.2 and v5 at the same time I think. We've had a hard time calling certain things finished for our first alpha. Once we have that figured out, we can announce a date :).
